### DISCUSSION TITLE
Errata on Page 11

### DISCUSSION THREAD

**Discussion by philippjohannsen** [rasbt/machine-learning-book#181] [General]
### Errata: Notational Conventions

Hi,

I noticed some inconsistent and confusing notation in the conventions on page 11 in the book. You find my suggestions in the attached PDF.
[Errata.pdf](https://github.com/rasbt/machine-learning-book/files/15471422/Errata.pdf)

**Comment by rasbt** [comment]
Thanks a lot for sending these. I agree with all your points. I just applied them to the Errata here: https://github.com/rasbt/machine-learning-book/blob/main/ERRATA/README.md#chapter-1 so that I don't forget to incorporate them in the next reprint update.
